docs/settings.rst

```diff
@@ -288,7 +288,14 @@ Basic settings
 
    A list of tags for Typogrify to ignore. By default Typogrify will ignore
    ``pre`` and ``code`` tags. This requires that Typogrify version 2.0.4 or
-   later is installed
+   later is installed.
+
+.. data:: TYPOGRIFY_OMIT_FILTERS = []
+
+   A list of Typogrify filters to skip. Allowed values are: ``'amp'``,
+   ``'smartypants'``, ``'caps'``, ``'initial_quotes'``, ``'widont'``. By
+   default, no filter is omitted (in other words, all filters get applied). This
```

Comment:
I wonder whether it would be better for this new setting to define which filters *should* be applied, with the default being the same: all filters are applied. For example, perhaps a default setting that looks something like:

```python
TYPOGRIFY_FILTERS = ["amp", "caps", "initial_quotes", "smartypants", "widont"]
```

I am not saying I am certain that the above approach is superior, but I figured now would be the time to consider it. What do you think?

Reply:
In general I favor "positive" settings like you suggest. For this particular one, I was on the fence. I thought that it perhaps made sense that all filters be applied by default without Pelican having to know what they are, or users having to know the full list when all they want is maybe remove one or two.

If in the future Typogrify was to get new filters, what behavior would be expected? With the "negative" setting, Pelican users automatically get the new filters as soon as they upgrade Typogrify, whether or not they have disabled some filters. With the "positive" setting, users who have kept the default will automatically get the new filters as soon as they upgrade both Typogrify and Pelican, but users who had specified a list of filters will need to add the new ones to their list (which is not necessary bad; those users might prefer to remain in control and not get unexpected new features).

`TYPOGRIFY_FILTERS` would seem more intuitive to use than `TYPOGRIFY_OMIT_FILTERS`, but I'm still on the fence. I'll go with the one you prefer. Just let me know!